Ports/Super-Mario/package.sh

```diff
@@ -17,5 +17,11 @@ configure() {
 
 install() {
     run mkdir -p "${SERENITY_INSTALL_ROOT}/opt/Super_Mario"
-    run cp -r uMario app.ico icon2.ico files "${SERENITY_INSTALL_ROOT}/opt/Super_Mario" 
+    run cp -r uMario files "${SERENITY_INSTALL_ROOT}/opt/Super_Mario"
+    if command -v convert >/dev/null; then
+        run convert "app.ico[0]" app-16x16.png
+        run convert "app.ico[1]" app-32x32.png
+        run objcopy --add-section serenity_icon_s="app-16x16.png" "${SERENITY_INSTALL_ROOT}/opt/Super_Mario/uMario"
+        run objcopy --add-section serenity_icon_m="app-32x32.png" "${SERENITY_INSTALL_ROOT}/opt/Super_Mario/uMario"
+    fi
```

Comment:
It might be nice to have this generalized and made available to all ports somehow.